<ui version="4.0" >
 <class>ActionTemplateSaveDialog</class>
 <widget class="QDialog" name="ActionTemplateSaveDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Сохранение шаблона действия</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible" >
         <bool>false</bool>
        </property>
        <widget class="QTreeView" name="treeItems" >
         <property name="autoScroll" >
          <bool>true</bool>
         </property>
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
        <widget class="CTableView" name="tblItems" >
         <property name="whatsThis" >
          <string comment="ура!" >список записей</string>
         </property>
         <property name="tabKeyNavigation" >
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ActionTemplateSaveDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>211</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel" >
     <x>94</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ActionTemplateSaveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>265</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel" >
     <x>404</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
